### DISCUSSION TITLE
Build swf?

### DISCUSSION THREAD

**Discussion by mxlgv** [FunkinCrew/Funkin#1569] [Q&A] [answered]
I read that haxe is capable of collecting files for a flash player. Is it possible to build the game as SWF?

**Comment by ghost** [comment]
Try looking at how Haxe builds like that and try it

**Comment by suprstarrd** [comment]
I tried building as-is, doesn't work too well. Will see if I can pull it off but I doubt it

**Comment by ghost** [comment]
ill do it later but im going somewhere and im using github for mobile this sucks

**Comment by Fefe543** [comment]
lime test swf

**Comment by TheHENOOB** [comment]
try `lime test flash` on any Command Line to run the game on flash, but you have to be aware that you probably gonna have to modify some lines of the game's code to run the game properly

**Reply by popopizza576** [reply]
it says couldnt do something idk what its on about

**Comment by kinoko87** [comment]
Probably only on flash targets. But since flash no longer is in support it might work weird? Idk.

**Comment by s-zenmode** [comment]
i tried doing it but it spits out some junk in console so it wont compile to swf as-is

**Reply by s-zenmode** [reply]
if you wanted to you could rewrite some stuff for flash support

**Comment by ninjamuffin99** [comment, marked as answer]
technically can work! just need to modify certain code and shit. and also flash player stuff is deprecated tho. remind me in like 3 years when [ruffle](https://ruffle.rs) has full support for emulating AS3 swf stuff!!!
